sql/core/src/test/resources/sql-tests/inputs/date.sql

```diff
@@ -104,6 +110,7 @@ select date '2001-09-28' - null;
 select '2011-11-11' - interval '2' day;
 select null - date '2019-10-06';
 select date '2001-10-01' - date '2001-09-28';
+select '2011-11-11 11:11:11' - date'2011-11-11';
 
```

Comment:
nit: we can put `select str - date'2011-11-11' from v2;` here.

Reply:
Done